source/macos/AdaptiveCards/AdaptiveCards/Renderers/ColumnSetRenderer.swift

```diff
@@ -45,6 +45,9 @@ class ColumnSetRenderer: BaseCardElementRendererProtocol {
             columnView.trailingAnchor.constraint(equalTo: wrappingView.trailingAnchor).isActive = true
             columnSetView.addView(wrappingView, in: gravityArea)
             BaseCardElementRenderer.shared.configBleed(collectionView: columnView, parentView: columnSetView, with: hostConfig, element: column, parentElement: columnSet)
+            let heightConstraint = columnView.heightAnchor.constraint(equalTo: columnSetView.heightAnchor)
+            heightConstraint.priority = .defaultLow
+            heightConstraint.isActive = true
         }
```

Comment:
Guess we need to add this [here](https://github.com/AdaptiveCards-Desktop/AdaptiveCards/blob/77539716da75c18ee01fe65d89a149a07572b5ff/source/macos/AdaptiveCards/AdaptiveCards/Renderers/ColumnSetRenderer.swift#L34-L35) also?